components/utils/ShapeList.js

```diff
@@ -290,19 +290,6 @@ const ShapeList = ({
     setFilteredShape(filterShape(copy, searchTerm));
   }, [searchTerm, shapes, sort]);
 
-  const handleSwicth = (shapeName) => {
```

Comment:
We don;t need this?

Reply:
Yes, it was an unused function that I missed.

Reply:
Ok @williamzhu17 , could you please test out the features one and conform all good to merge?

Reply:
@atapas just tested all the features and they are all working. should be ready to merge